src/styles/app/47-external-markdown.css

```
@@ -1,5 +1,12 @@
 .externalMarkdownWindow {
 	--external-markdown-chrome-height: 56px;
+	--external-markdown-status-height: 28px;
+	--external-markdown-editor-min-height: calc(
+		100vh -
```

Stylelint (17.14.1)

[error] Unexpected newline after "-" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)